.travis.yml

```diff
@@ -28,6 +33,33 @@ script:
   - pushd cherry_picker
   - pytest cherry_picker/test.py -v
   - popd
+- &deploy-base
+  stage: Publish dists to PYPI
+  python: "3.6-dev"
+  script:
+  - flit build
+  # Add an empty setup.py stub, because pypi provider always calls it
+  - touch setup.py
+  deploy:
+    provider: pypi
+    # `skip-cleanup: true` is required to preserve binary wheel and sdist,
+    # built by during `install` step above.
+    skip-cleanup: true
+    # `skip-existing: true` is required to skip uploading dists, already
+    # present in PYPI instead of failing the whole process.
+    # This happenes when other CI (AppVeyor etc.) has already uploaded
+    # the very same dist (usually sdist).
+    skip-existing: true
+    user: Mariatta  # TODO: consider having a separate "uploader user"
```

Comment:
@Mariatta you might want to set up a separate account with "Maintainer" level privileges in PYPI to use instead of your own, which is more dangerous if it gets compomised.

Recently, we've started replacing our flows to use a deployment-dedicated user in aio-libs as well: https://github.com/aio-libs/aiohttp/blob/master/.travis.yml#L84

Reply:
@Mariatta I've just realized that blurb is deployed by @brettcannon and your account is not linked to that dist. Will you fix this by having an account with access to publish both or should I separate deployment step to have different deployer users?